src/hotspot/cpu/ppc/stubGenerator_ppc.cpp

```diff
@@ -3786,24 +3843,38 @@ class StubGenerator: public StubCodeGenerator {
     VectorSRegister vec_special_case_offset = VSR4;
     VectorSRegister pack_permute            = VSR5;
 
-    // Variables for lookup
-    // VR
+    // P10+ VSR lookup constants
+    VectorSRegister table_32_47             = VSR0;
+    VectorSRegister table_48_63             = VSR1;
+    VectorSRegister table_64_79             = VSR2;
+    VectorSRegister table_80_95             = VSR3;
+    VectorSRegister table_96_111            = VSR4;
+    VectorSRegister table_112_127           = VSR6;
+
+    // Data read in and later converted
     VectorRegister  input                   = VR6;
+    // Variable for testing Base64 validity
+    VectorRegister  non_match               = VR10;
+
+    // P9 VR Variables for lookup
     VectorRegister  higher_nibble           = VR7;
     VectorRegister  eq_special_case_char    = VR8;
     VectorRegister  offsets                 = VR9;
-    VectorRegister  non_match               = VR10;
 
-    // VSR
+    // P9 VSR lookup variables
     VectorSRegister bit                     = VSR6;
     VectorSRegister lower_nibble            = VSR7;
     VectorSRegister M                       = VSR8;
 
+    // P10+ VSR lookup variables
```

Comment:
"P10+" should better be "P10 or above" (or something grammatically correct) because it appears like an enhanced version of P10, like P7+.  Although IBM didn't released "+" models after P8, they have manufactured with reduced semiconductor process and apply enhancements, including larger caches, higher clock frequency, etc.

Reply:
Good point.  Will fix.

Reply:
I believe I have resolved this with my latest commit to this PR.  Please double check.